requirements.txt

```diff
@@ -1,20 +1,99 @@
-accelerate
-argh
-dacite
-demucs
-diffusers>=0.9.0
-flask
-flask_cors
-numpy
-pillow>=9.1.0
-plotly
-pydub
-pysoundfile
-scipy
-soundfile
-sox
-streamlit>=1.10.0
-torch
-torchaudio
-torchvision
-transformers
+accelerate==0.16.0 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+altair==4.2.2 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+antlr4-python3-runtime==4.9.3 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+argh==0.27.2 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+attrs==22.2.0 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+backports-zoneinfo==0.2.1 ; python_version >= "3.8" and python_version < "3.9"
+blinker==1.5 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+cachetools==5.3.0 ; python_version >= "3.8" and python_version < "3.12" and python_full_version != "3.9.7"
+certifi==2022.12.7 ; python_version >= "3.8" and python_version < "3.12" and python_full_version != "3.9.7"
+cffi==1.15.1 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+charset-normalizer==3.0.1 ; python_version >= "3.8" and python_version < "3.12" and python_full_version != "3.9.7"
+click==8.1.3 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+cloudpickle==2.2.1 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+colorama==0.4.6 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12" and platform_system == "Windows"
+cython==0.29.33 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+dacite==1.8.0 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+decorator==5.1.1 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+demucs==4.0.0 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+diffq==0.2.3 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+diffusers==0.12.1 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+dora-search==0.1.11 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+einops==0.6.0 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+entrypoints==0.4 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
+filelock==3.9.0 ; python_version >= "3.8" and python_full_version != "3.9.7" and python_version < "3.12"
```

Comment:
I'm confused why these are all locked, when there's already poetry.lock? It seems like this requirements file won't play nice with anyone else